datasets/ONS-UK-trade-in-goods-by-industry-country-and-commodity/concat_main.py

```diff
@@ -0,0 +1,118 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.6.0
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# UK trade in goods by industry, country and commodity, exports and imports
+
+import pandas as pd
+import json
+from gssutils import *
+
+pd.options.mode.chained_assignment = None 
+
+cubes = Cubes("info.json")
+
+# +
+with open ('info.json') as file:
+    info = json.load(file)
+
+landingPage = info['landingPage']
+landingPage
+
+# +
+scraper1 = Scraper(landingPage[0])
+scraper2 = Scraper(landingPage[1])
+
+scraper1.dataset.family = info['families']
+scraper1
+# -
+
+scraper2
+
+distribution1 = scraper1.distribution(latest=True)
+distribution2 = scraper2.distribution(latest=True)
+distribution1
+
+distribution2
+
+tabs1 = distribution1.as_databaker()
+tabs2 = distribution2.as_databaker()
+tabs = tabs1 + tabs2
+
+title1 = distribution1.title
+title2 = distribution2.title
+
+# +
+trace = TransformTrace()
+title = distribution1.title + ' and imports' 
+columns = ['ONS Partner Geography', 'Period','Flow','CORD SITC', 'SIC 2007', 'Measure Type','Value','Unit', 'Marker']
+
+for tab in tabs:
+    if tab.name not in ['tig_ind_ex', 'tig_ind_im']:
+        continue
+    print(tab.name)
+    
+    trace.start(title1, tab, columns, distribution1.downloadURL)
+    trace.start(title2, tab, columns, distribution2.downloadURL)
+    
+    if tab.name == 'tig_ind_ex':
+        flow = 'exports'
+    elif tab.name == 'tig_ind_im':
+        flow = 'imports'
+    print(flow)   
+ 
+    country = tab.filter('country').fill(DOWN).is_not_blank()
```

Comment:
Good idea to add tracing comments for each dimension so that we can see where each value has been found in the dataset. E.g:
`trace.country("Values taken from the cell found by filtering entire tab for 'country' (A1) downwards")`